### PR TITLE
[express-actuator] update to version 1.8

### PR DESCRIPTION
**Pull request by rcruzper**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1799,7 +1799,7 @@
 /types/expo-mixpanel-analytics/                         @martintreurnicht
 /types/expo__status-bar-height/                         @dawnmist
 /types/express/                                         @borisyankov @CMUH @puneetar @dfrankland
-/types/express-actuator/                                @etruta
+/types/express-actuator/                                @etruta @rcruzper
```
Comment (peterblazejewicz) [inline .github/CODEOWNERS:1802]:
```suggestion
/types/express-actuator/                                @ethanrubio
```

Comment (rcruzper) [reply .github/CODEOWNERS:1802]: oh, sorry I didn't understand the documentation, fixed.

**Comment by orta** [comment]
This kinda needs a review from @etruta WRT adding another reviewer.

**Review by etruta — APPROVED** [review]
Sorry guys, I was offline the last few days.

**Comment by rcruzper** [comment]
Ready to merge

**Pull request merged**